include/picongpu/particles/pusher/particlePusherFree.hpp

```diff
@@ -52,6 +54,17 @@ namespace picongpu
                 using MomType = momentum::type;
                 MomType const mom = particle[momentum_];
 
+                const auto bField = functorBField(pos);
+                const auto eField = functorEField(pos);
```

Comment:
BUG: The guard size of the pusher must be adjusted if we access the fields to interpolate the fields!

Reply:
False alert: The guard size depends on the interpolation function. If a probe has defined interpolation the guard is derived correctly.

```
    using ParticleFlagsProbes = MakeSeq_t<
        particlePusher<particles::pusher::Probe>,
        shape<UsedParticleShape>,
        interpolation<UsedField2Particle>>;
```